src/language/syntax.rkt

```diff
@@ -168,6 +168,7 @@
 
 
 (define-syntax-rule (op$ o a ...) (Op 'o (list a ...)))
+(define-syntax-rule (app$ cl a ...) (App-Code cl (list a ...)))
```

Comment:
This should probably be named `appc$` or `app-code$` to distinguish it from a macro that creates the `App` form.

Reply:
Done.